test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMemoryMXBean.java

```diff
@@ -148,20 +147,6 @@ protected void tearDown() throws Exception {
 		}
 	}
 
-	static class ClassForTestMaxHeapSize implements Task {
-		@Override
-		public void run() throws Exception {
-			System.setSecurityManager(new SecurityManager());
-			MemoryMXBean bean = (MemoryMXBean)ManagementFactory.getMemoryMXBean();
-			Thread.sleep(2000);
-			long size = bean.getMinHeapSize();
-			bean.setMaxHeapSize(size + 1024);
-			if (size + 1024 != bean.getMaxHeapSize()) {
-				throw new RuntimeException("not equal");
-			}
```

Comment:
Is this a duplicated sub-test somewhere, otherwise, pls keep original version, and create one w/o  `System.setSecurityManager(new SecurityManager())`.

Reply:
I don't think this class is used at all